frontend/src/app/App.scss

```diff
@@ -64,6 +64,14 @@ body,
   }
 }
 
+// giving a fixed width and height of the icon for app launcher
+.application-launcher-menu-item-icon {
+  .pf-v5-c-menu__item-icon {
+    width: 1.5rem;
+    height: 1.5rem;
+  }
+}
+
```

Comment:
Replacing the deprecated application launcher with a dropdown, noticed that the icons for dropdown items were not rendering. This is because the deprecated application launcher item icons had a fixed width and height specified,  That is the reason added css style